lib/emoji/extractor.rb

```diff
@@ -23,6 +23,12 @@ def extract!
         hexes = ttf_name.split('_').map { |n| n.gsub(/^u/, '').downcase }
 
         if emoji_char_codes.include?(hexes.first.to_i(16))
+          # When an emoji supports skin-tone modifiers, all variants--including
+          # the unmodified "base" emoji--are suffixed with a "." + a number:
+          # 0 for the base, then 1-5 for Fitzpatrick type
+          # (https://en.wikipedia.org/wiki/Fitzpatrick_scale) 2-6. We remove the
```

Comment:
`1f44b.1.png` corresponds to :wave: in shade Fitzpatrick type II, `1f44b.2.png` to Fitzpatrick type III, and so on.